app_legacy/Helpers/render/site-award.php

```diff
@@ -132,15 +136,22 @@ function RenderAwardGroup($awards, $title): void
         $counters = RenderCounter($icon, $text, $numItems, $numHidden);
     }
 
+    $fadeEnabled = count($awards) > 50 ? 'fade-enabled' : '';
     echo "<div id='" . strtolower(str_replace(' ', '', $title)) . "'>";
     echo "<h3 class='flex justify-between gap-2'><span class='grow'>$title</span>$counters</h3>";
-    echo "<div class='component flex flex-wrap justify-start gap-2'>";
+    echo "<div class='relative'>";
+    echo "<div onscroll='foo(event)' class='component flex flex-wrap justify-start gap-2 max-h-[340px] lg:max-h-[640px] lg:scrollbar-none overflow-y-auto transition'>";
```

Comment:
The fade effect should not block user scrolling. There is a Tailwind class for this, just need to find it